CODE/Sources/PennyAPI/AddCoin/AddCoinLambdaHandler.swift

```diff
@@ -68,14 +68,20 @@ struct AddCoins: LambdaHandler {
                 source: product.source,
                 reason: product.reason)
             
-            let message = try await userService.addCoins(with: coinEntry, to: user)
-            response = APIGatewayV2Response(statusCode: .ok, body: message)
+            let coinResponse = try await userService.addCoins(
+                with: coinEntry,
+                fromDiscordID: product.from,
+                to: user
+            )
+            let data = try JSONEncoder().encode(coinResponse)
+            let string = String(data: data, encoding: .utf8)
+            response = APIGatewayV2Response(statusCode: .ok, body: string)
         }
         catch UserService.ServiceError.failedToUpdate {
             response = APIGatewayV2Response(statusCode: .notFound, body: "ERROR- The user in particular wasn't found.")
         }
         catch let error {
-            response = APIGatewayV2Response(statusCode: .badRequest, body: "ERROR-\(error.localizedDescription)")
+            response = APIGatewayV2Response(statusCode: .badRequest, body: "ERROR- \(error.localizedDescription)")
```

Comment:
Is there supposed to be a space here?

Reply:
I just looked at the log that was above it and it had space so I thought maybe this should have a space too 😅. Otherwise I guess it doesn't really matter?

Reply:
Ah, mistake on my end then haha. But it doesn't matter that much tbf, we should just be consistent, that's all